src/portal/src/app/distribution/distribution-setup-modal/distribution-setup-modal.component.ts

```diff
@@ -196,6 +208,14 @@ export class DistributionSetupModalComponent implements OnInit {
       if ( this.model.endpoint !== this.originModelForEdit.endpoint) {
         return true;
       }
+      // tslint:disable-next-line:triple-equals
+      if ( this.model.enabled != this.originModelForEdit.enabled) {
+        return true;
+      }
+      // tslint:disable-next-line:triple-equals
+      if ( this.model.insecure != this.originModelForEdit.insecure) {
```

Comment:
Why donot use '!=='  and delete  '// tslint:disable-next-line:triple-equals'

Reply:
"", null and undefined need to be regarded as same value

Reply:
I don’t think he will have undefined value. If it does, it means there are other errors.

Reply:
I don’t think he will have undefined value. If it does, it means there are other errors.

Reply:
Supposed that i created a  policy by call api   and   did not set insecure property,  when i edit this policy , and set its insecure to false.    so its original value is 'undefined'  and current value is 'false'. if use '!==', we get false, but infact , we should regard these two value  are equal